docs/contributing/minutes.md

```diff
@@ -4,6 +4,94 @@ title: Team Minutes
 sidebar_label: Team Minutes
 ---
 
+## 05/01/21
+
+### Present
+
+Ali, Mitch, Chris, Sally, Greg
+
+### Minutes
+
+- Fundamentals
+  - Sally worked out the timings (incl PD things): https://docs.google.com/spreadsheets/d/1nKm9cwfYzyPGt2W_P9GQPPavmnD5zFEeTTQTdOgpRVg/edit#gid=0
+  - Red is immovable stuff, otherwise divided into planned work
+  - Blockly
+    - Greg's planned schedule
+      - Week 1 - Code.org, intro blockly, intro user stories
+      - Week 2 - ? [missed this, sorry]
```

Comment:
I unfortunately missed what the proposal here was - @gregdyke @SallyMcGrath do you remember what we discussed?

Reply:
Nope!